NHLGames/French.Designer.vb

```diff
@@ -847,6 +847,15 @@ Namespace My.Resources
             End Get
         End Property
         
+        '''<summary>
+        '''  Looks up a localized string similar to Caméra Sedins.
+        '''</summary>
+        Friend Shared ReadOnly Property lblSedinsCam() As String
```

Comment:
no new resource, star cam is the one to use, like ovechkin and crosby cam, or goalie cam